src/wp-includes/ms-files.php

```diff
@@ -17,8 +18,6 @@
 
 ms_file_constants();
```

Comment:
Looking into `ms_file_constants()` there is a small argument to be made for a new `MS_FILES_REQUEST` constant to be added inside it instead of at the top of `ms-files.php`, but ultimately the constants that get defined inside `ms_file_constants()` are optional and related to media serving, not the WordPress environment itself, so I don't think that needs to change.

Reply:
That's a good point, I'd be happy either way.